emojivoto-web-app/js/setup_dev_env.sh

```diff
@@ -4,7 +4,27 @@
 CONTAINER_ID=''
 OS=''
 ARCH=''
-MOUNT_VOLUME=''
+MOUNT_VOLUME_LOCAL=''
+USE_TELEMETRY=
+ACTIVITY_REPORT_TYPE='INTERMEDIATE_CLOUD_TOUR'
+
+use_telemetry() {
+    USE_TELEMETRY=true
+}
+
+send_telemetry() {
+    if [ $USE_TELEMETRY = true ]; then
+        action=$1
+        ambassador_cloud_url="https://auth.datawire.io"
+        application_activities_url="${ambassador_cloud_url}/api/applicationactivities"
+        curl -X POST \
+          -H "X-Ambassador-API-Key: $AMBASSADOR_API_KEY" \
+          -H "Content-Type: application/json" \
+          -d '{"type": "'$ACTIVITY_REPORT_TYPE'", "extraProperties": {"action":"'"$action"'","os":"'"$OS"'","arch":"'"$ARCH"'"}}' \
```

Comment:
We will need to update the DBT to be able to read the `os` and `arch` easily right?

Reply:
We can use  `JSON_EXTRACT(extra_properties, "$.os")`, if we do not filter by the value we should not have big performance issue.  If we find the performance is not good, we can add it later.

Reply:
Yes, but my understanding is that is expensive querying by json in Big Query.

Reply:
Let's report it and if we decided that we want to filter this data, then we can go and update de DBT.